Source/WebCore/platform/mediastream/gstreamer/GStreamerVideoCapturer.h

```diff
@@ -48,7 +48,8 @@ class GStreamerVideoCapturer final : public GStreamerCapturer {
     void setSinkVideoFrameCallback(SinkVideoFrameCallback&&);
 
 private:
-    bool setSize(int width, int height);
+    bool setSize(const IntSize&);
+    const IntSize& size() const { return m_size; }
```

Comment:
why do you need this method? It's private, why not accessing directly the `m_size`?

Reply:
It's used by `MockRealtimeVideoSourceGStreamer` which isn't a capturer.